2022-01-14 13:40:01,080 INFO    MainThread:413829 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 13:40:01,080 INFO    MainThread:413829 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:40:01,080 INFO    MainThread:413829 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 13:40:01,081 INFO    MainThread:413829 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_134001-295ddx9o/logs/debug.log
2022-01-14 13:40:01,081 INFO    MainThread:413829 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_134001-295ddx9o/logs/debug-internal.log
2022-01-14 13:40:01,081 INFO    MainThread:413829 [wandb_init.py:init():404] calling init triggers
2022-01-14 13:40:01,081 INFO    MainThread:413829 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 13:40:01,082 INFO    MainThread:413829 [wandb_init.py:init():460] starting backend
2022-01-14 13:40:01,082 INFO    MainThread:413829 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 13:40:01,088 INFO    MainThread:413829 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 13:40:01,091 INFO    MainThread:413829 [backend.py:ensure_launched():221] started backend process with pid: 413889
2022-01-14 13:40:01,092 INFO    MainThread:413829 [wandb_init.py:init():469] backend started and connected
2022-01-14 13:40:01,094 INFO    MainThread:413829 [wandb_init.py:init():533] updated telemetry
2022-01-14 13:40:01,103 INFO    MainThread:413829 [wandb_init.py:init():563] communicating current version
2022-01-14 13:40:01,461 INFO    MainThread:413829 [wandb_init.py:init():568] got version response 
2022-01-14 13:40:01,461 INFO    MainThread:413829 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 13:40:01,636 INFO    MainThread:413829 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 13:40:01,688 INFO    MainThread:413829 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 13:40:01,689 INFO    MainThread:413829 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 13:40:01,689 INFO    MainThread:413829 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 13:40:01,690 INFO    MainThread:413829 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 13:40:01,690 INFO    MainThread:413829 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 13:40:10,401 INFO    MainThread:413829 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 13:40:10,402 INFO    MainThread:413829 [wandb_run.py:_restore():1752] restore
2022-01-14 13:40:12,783 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:40:13,031 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 13:40:13,667 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 9500
}

2022-01-14 13:40:13,770 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 9500
}

2022-01-14 13:40:13,872 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9500
  total_bytes: 9500
}

2022-01-14 13:40:13,973 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9500
  total_bytes: 9500
}

2022-01-14 13:40:14,074 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9500
  total_bytes: 9500
}

2022-01-14 13:40:14,175 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9500
  total_bytes: 9500
}

2022-01-14 13:40:14,276 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9500
  total_bytes: 9500
}

2022-01-14 13:40:15,437 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9500
  total_bytes: 9500
}

2022-01-14 13:40:15,808 INFO    MainThread:413829 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9500
  total_bytes: 9500
}
local_info {
}

2022-01-14 13:40:16,891 INFO    MainThread:413829 [wandb_run.py:_append_files():2180] logging synced files
